tests/cmake/default

```diff
@@ -31,12 +33,17 @@ while ( <STDIN> )
 	$skip=1;
     }
 
+    if ($_ =~ m/^\s*An error occurred in line/)
+    {
+  $line = "(line in output replaced by default.sh script)\n";
```

Comment:
Indentation seems to be weird

Reply:
The indentation of the whole file was weird. Now fixed. I don't understand why the tests failed (the script works fine on my machine). Let's see if they pass now.